src/assets/stylesheets/variables/_responsive.scss

```diff
@@ -0,0 +1,6 @@
+$screen-xs-max: 767px;
```

Comment:
Please add the following comment on the very top of the file:

```scss
/* - BS3 variables - */
```

This will help our designer to identify the source of these variables.